uefi/ast.go

```diff
@@ -0,0 +1,6 @@
+package uefi
+
+type ASTNode interface {
+	Apply(v Visitor) error
+	ApplyChildren(v Visitor) error
```

Comment:
ApplyRecursive? Dunno, I don't feel strongly.

Reply:
It only applies to the children, so I feel the name is apt.